utils/node_local_test.py

```diff
@@ -1478,6 +1481,27 @@ def test_uns_basic(self):
         if dfuse.stop():
             self.fatal_errors = True
 
+    def test_cont_copy(self):
```

Comment:
Minor suggestion - only if you need to push again:
There are two tools:
```
daos container copy # Copy cont->cont with Obj API
daos filesystem copy # Copy cont<->POSIX with DFS API
```
So the name `test_cont_copy` might be confusing if ever there is a need to to add a `daos container copy` test here.

Reply:
@daltonbohning @ashleypittman the Obj API version should be 'daos cont clone', so we could just use that as the name.

Reply:
I'lll add that in a future PR.